packages/test-runner/src/index.ts

```diff
@@ -159,3 +162,18 @@ registerFixture('tmpDir', async ({}, test) => {
   await removeFolderAsync(tmpDir).catch(e => {});
 });
 
+registerFixture('generateAssetFileName', async ({}, runTest, info) => {
```

Comment:
`generateAssetFileName` is a mouth full. Also, `asset` is not ideal for a generated file, it is more of a file that is version controlled. We already have `outputDir`, so maybe `outputFile()`?

Reply:
Good point 👍